manim/mobject/coordinate_systems.py

```diff
@@ -52,8 +52,8 @@ def __init__(
                 1.0,
             ]
 
-        self.x_range = np.array(x_range)
-        self.y_range = np.array(y_range)
+        self.x_range = x_range
+        self.y_range = y_range
```

Comment:
What was the reasoning for removing `np.array`? I think numpy generally offers a speed increase (not sure how inconsequential it may or may not be here)

Reply:
If we would perform any numerical computations with the numbers, sure. But the list is literally only used for storing the three values, such that other parts of the code can access it. There is no need to use numpy for that (and also no realistic performance increase to be expected).